scalameter-core/src/main/scala/org/scalameter/utils/ClassPath.scala

```diff
@@ -8,51 +8,73 @@ import org.apache.commons.lang3.SystemUtils
 
 
 
-object ClassPath {
-  /** Returns the default classpath string.
+class ClassPath private (val paths: List[String]) extends Serializable {
```

Comment:
To be completely platform-independent, maybe you want to have a list of `File`s here?